evap/static/scss/evap.scss

```diff
@@ -147,3 +147,67 @@ td.icon {
 @media print {
     @import "print";
 }
+
+.notebook-animation {
+    transition: width .0s !important;
+}
+
+.notebook-div {
+    height: 100%;
+    display: flex;
+	flex-direction: column;
+	justify-content: flex-start;
+	align-items: stretch;
+	align-content: stretch;
+}
+
+.notebook-button {
+    flex: 0 0 auto;
+}
+
+@media screen and (min-aspect-ratio: $notebook-break) {
+    .notebook-margin {
+        max-width: 75vw;
+        margin-left: 25vw;
+    }
+
+    .notebook-container {
+        width: 25vw;
+        padding-left: $spacer;
+        padding-bottom: $spacer;
+        padding-top: $spacer;
+    }
+
+    .notebook-card {
+        // substracting 3 * Margins 2x card 1x breadcrumb
+        height: calc((100vh - #{$navbar-brand-height}) - (#{$footer-height} + ((#{$spacer} * 1.5) * 3)));
```

Comment:
I either don't understand the comment or the computation -- there is no `2 * ` in there?